lib/flight.js

```diff
@@ -124,3 +124,13 @@ Flight.prototype.rightFlip = function() {
 Flight.prototype.wave = function() {
   return this.connection.animate("wave", 750);
 };
+
+/**
+ * Tells the drone to perform horizontal trimming.
+ *
+ * @return {null}
+ * @publish
+ */
+Flight.prototype.ftrim = function() {
```

Comment:
@deadprogram cool, thanks Ron, I'll update the code and the pull request as soon as I can.

Out of curiosity, why are some methods explicitly (re)defined in the flight.js file?